Automattic/vip-go-mu-plugins
# WP-CLI: Unblock `wp db tables` command

## Description
This unblocks the `wp db tables` CLI command. 

⚠️ https://github.com/Automattic/vip-go-mu-plugins/pull/4376 needs to be merged first
## Changelog Description

### Plugin Updated: Wp_Cli_Db

WP-CLI: Unblock `wp db tables` command

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test
1) Check out PR
2) Run `wp db tables` and expect it to run
